### PR TITLE
Also create a GitHub release page upon doing a release

### PR DESCRIPTION
**Pull request by rwaffen**
sadly https://github.com/voxpupuli/gha-create-a-github-release/pull/1 does not work. so simply put the release.yml here, like the labeler.yml.

### CODE REVIEW

**Comment by bastelfreak** [comment]
@rwaffen looks good to me, could you run this on our puppet-example module and release it? So we know that it works properly.

**Comment by rwaffen** [comment]
can do

File: moduleroot/.github/release.yml.erb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+# https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes
+
+changelog:
+  exclude:
+    labels:
+      - skip-changelog
```
Comment (bastelfreak) [inline moduleroot/.github/release.yml.erb:7]: I think `modulesync` should be added here as well? Or everything from https://github.com/voxpupuli/voxpupuli-release/blob/master/lib/voxpupuli/release/rake_tasks.rb#L140 ?

File: moduleroot/.github/release.yml.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+  categories:
```
Comment (bastelfreak) [inline moduleroot/.github/release.yml.erb:9]: should our `bug` label get its own category?

File: moduleroot/.github/release.yml.erb (hunk 1 above)
Comment (rwaffen) [reply moduleroot/.github/release.yml.erb:7]: yeah sure, can add all of this.

File: moduleroot/.github/release.yml.erb (hunk 2 above)
Comment (rwaffen) [reply moduleroot/.github/release.yml.erb:9]: makes sense to have a bug category 🤔

**Comment by rwaffen** [comment]
workflow tested with a release... https://github.com/voxpupuli/puppet-example/releases/tag/v0.4.0

now I will do the requested changes

**Comment by TheMeier** [comment]
nice improvement :)

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**